docs/deploy.md

```diff
@@ -0,0 +1,131 @@
+# Procedure for releasing new AMY version
+
+1. Close milestone using [GitHub UI](https://github.com/swcarpentry/amy/milestones).
+
+1. Follow procedure for deployment on production.
+
+1. Write down release notes and publish it on [Piotr's blog](http://piotr.banaszkiewicz.org/). Release notes should have two sections: new features and bugfixes, each one enumerating changes and mentioning their authors. Base your work on the list of all closed pull request for given milestone, available on GitHub.
```

Comment:
1. "all closed pull request**s**"
2. Can you split long lines? It will be easier to comment on single lines.